### ISSUE TITLE
Get kicked out of Plex on Roku

### ISSUE THREAD

**Issue by shopgirl284** [mikew/ss-plex.bundle#22]
Certain videos give an error and when I try to choose play with them it completely kick me out of Plex. I think it is probably just a bad video, but would like to see if you could find a way to add an exception for this so I do not have to go through reentering Plex and all the layers of SSPlex everytime I encounter one of these videos. I usually encounter at least one each day.

Here is the code it produces in the system log http://pastebin.com/x3JXceMi

I will try to keep an record of the errors and see if I can figure out if there is a simple exception line that can be added to the URL service for it.

**Comment by mikew** [comment]
If you try other billion upload sites does the error keep happening?
- Mike Wyatt

**Comment by shopgirl284** [comment]
The good news is that I have not had an issue with this for any shows I tried to play lately, the resolution of the issue seemed to correspond with your latest update, but I cannot be sure. Due to the issues lately, I have not used SSPlex as much this week as I was in the week prior. 

To make sure it was resolved, I just went back to the particular episode that caused the error in the log above and tried all the billion upload sources available for it and all worked fine.

I also just tested a one more I had found (ink masters season 2 episode 3) that was kicking me out of Plex and that one also works now. It kicked me out for sockshare, putlocker, and first two billionsupload, but movereel.com worked. (One of the billion upload sources would not play, but I realize that happens, the issue was it kicking me out of Plex.)

Just glad that it is working again
